stdlib/imaplib.pyi

```diff
@@ -171,5 +171,5 @@ class _Authenticator:
 
 def Internaldate2tuple(resp: bytes) -> time.struct_time: ...
 def Int2AP(num: int) -> str: ...
-def ParseFlags(resp: str) -> tuple[str, ...]: ...
+def ParseFlags(resp: str) -> tuple[bytes, ...]: ...
```

Comment:
`ParseFlags("hi")` gives an error to me. The argument must be `bytes` too.

```suggestion
def ParseFlags(resp: bytes) -> tuple[bytes, ...]: ...
```

Reply:
Fixed.